### ISSUE TITLE
Optionally run the tests

### ISSUE THREAD

**Issue by gelisam** [gelisam/cabal-rangefinder#7]
Sometimes, the package compiles fine on a given version, but the tests don't run. Currently, we don't run the tests at all. We should. But since running the tests often take much longer than building, we should make them optional. Perhaps we could also allow an external verification script to be ran?

**Comment by Wizek** [comment]
Does `cabal-rangefinder` mutate the current working directory? If so, then this should be rather straightforward to implement, as follows, wouldn't it?
- If the user specifies `--enable-tests` for `cabal-rangefinder`, e.g. `cabal-rangefinder my.cabal --enable-tests`
  - Then run `cabal install --enable-tests && cabal test` on each iteration.
  - Else run as we do currently.

**Comment by gelisam** [comment]
Yes, I agree that it should be straightforward. Would you like to take a stab at it? I'm rather busy with other Haskell projects at the moment.
